### ISSUE TITLE
Attribute Mask(VaInput) has an error with touch event

### ISSUE THREAD

**Issue by EugenGedroyc** [epicmaxco/vuestic-ui#3861]
**Vuestic-ui version:** 1.8.0 (also tested in 1.7.6 before the update)

**Steps to reproduce**
Create any VaInput with attributes: 'clearable' and ':mask'
```
 <va-input
    v-model="transportNumber"
    clearable
    :mask="{ blocks: [1, 3, 2] }"
/>
```
1. Switch to mobile version(dev tools: ctrl+shift+m)
2. Type at least one character
3. In VaInput, touch the button with the "clearable function"(a cross in a circle by default)

**What is the expected behavior?**

Same behaviour as with the click event 

**What is the current behavior?**

The data in a VaInput is not cleared until you make a touch in another VaInput or this VaInput 

**Other information**

**Comment by m0ksem** [comment]
Clearable works for me on my phone and in mobile devtools. Maybe it doesn't work only if there is musk?

**Comment by EugenGedroyc** [comment]
Yes, this bug only with mask. correction: "The data in a VaInput is not cleared until you make a touch in another VaInput or this VaInput"

**Comment by m0ksem** [comment]
Closing, because we used different mask system from now.
